strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/maven/MavenArtifactController.java

```diff
@@ -213,7 +218,48 @@ public void download(@ApiParam(value = "The storageId", required = true)
 
         logger.debug("Download succeeded.");
     }
-
+    
+    @ApiOperation(value = "Used to retrieve headers of an artifact", position = 2)
+    @ApiResponses(value = { @ApiResponse(code = 200, message = ""),
+                            @ApiResponse(code = 400, message = "An error occurred.") })
```

Comment:
`@ApiResponse(code = 400, message = "An error occurred.")`

Are you sure you return `400` somewhere below ?